src/api/app/validators/scm_webhook_event_validator.rb

```diff
@@ -4,6 +4,10 @@ class ScmWebhookEventValidator < ActiveModel::Validator
 
   ALLOWED_PULL_REQUEST_ACTIONS = ['closed', 'opened', 'reopened', 'synchronize'].freeze
   ALLOWED_MERGE_REQUEST_ACTIONS = ['close', 'merge', 'open', 'reopen', 'update'].freeze
+  NOT_ALLOWED_PULL_REQUEST_ACTIONS = ['assigned', 'auto_merge_disabled', 'auto_merge_enabled', 'converted_to_draft',
+                                      'edited', 'labeled', 'locked', 'ready_for_review', 'review_request_removed',
+                                      'review_requested', 'unassigned', 'unlabeled', 'unlocked'].freeze
+  NOT_ALLOWED_MERGE_REQUEST_ACTIONS = ['approved', 'unapproved'].freeze
```

Comment:
Do we really need to specify the list of not allowed ones? Is it not enough to ask if the incoming event IS NOT in the ALLOWED_*_ACTIONS?